xbmc/interfaces/json-rpc/FileItemHandler.cpp

```diff
@@ -359,7 +359,7 @@ void CFileItemHandler::HandleFileItem(const char *ID, bool allowFile, const char
     if (item->HasEPGInfoTag())
       FillDetails(item->GetEPGInfoTag().get(), item, fields, object, thumbLoader);
     if (item->HasPVRRecordingInfoTag())
-      FillDetails(item->GetPVRRecordingInfoTag(), item, fields, object, thumbLoader);
+      FillDetails(item->GetPVRRecordingInfoTag().get(), item, fields, object, thumbLoader);
```

Comment:
declaration: <code>void FillDetails(const ISerializable *info, ...</code>

BTW: Did it the same way as for epginfotag, were this was fine for you.